lab/homework/index.js

```diff
@@ -0,0 +1,177 @@
+"use strict";
```

Comment:
`'use strict'` is implied automatically by the ES Module Loader. It's needed only for CommonJS modules.